Extensions/DialogueTree/dialoguetools.js

```diff
@@ -725,3 +725,16 @@ gdjs.dialogueTree.loadState = function(inputVariable) {
     console.error('Failed to load state from variable:', inputVariable, e);
   }
 };
+
+/**
+ * Clear the current State of the Dialogue Parser.
+ */
+gdjs.dialogueTree.clearState = function() {
+  try {
+    gdjs.dialogueTree.runner.visited = {};
+    gdjs.dialogueTree.runner.variables.data = {};
+    });
+  } catch (e) {
+    console.error('Failed to clear yarn state:', e);
```

Comment:
Is this possible for the previous assignments to fail?

Reply:
I think I can get rid of the try/catch here. Left it just in case

Reply:
removed it now